ocs_ci/templates/CSI/rbd/pod_block.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
is it possible to utilize the existing pod.yaml and after creating a dict out of it in ocs_ci/ocs/constants.py, change its items to support raw block?
This should be done in helpers.create_pod()

Reply:
Updated create_pod()